server/controllers/lecture-controller.js

```diff
@@ -188,15 +191,16 @@ exports.getScheduledLectures = async (req, res) => {
   knex
     .select(
       { id: "l.id" },
-      { name: "l.name" },
+      //{ name: "l.name" },
```

Comment:
Usually it's seen as a bad practice to commit files containing commented lines of code.